docs/content/demo/lorenz_tesseract.md

```diff
@@ -0,0 +1,23 @@
+# Building the JAX Solver Tesseract for Lorenz-96
```

Comment:
We should write somewhere here that first one initializes the jax recipe, then fills with the following -- otherwise it might not be clear from this page or demo.md where differentials are coming from

Reply:
Yes we'll iterate more on this page after this PR is merged. CC @yongquan-qu, your move!